optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/roster/RosterRestServiceImpl.java

```diff
@@ -274,4 +276,9 @@ public void updateShiftsOfRoster(Roster newRoster) {
                     ? null : employeeIdMap.get(shift.getEmployee().getId()));
         }
     }
+
+    // TODO: Make this a REST method?
```

Comment:
Yes, once we build the roster state configuration screen (of which most fields will be read-only)